### DISCUSSION TITLE
Importance of PDB for Rook

### DISCUSSION THREAD

**Discussion by zeaone** [rook/rook#11187] [Q&A] [answered]
The usage scenarios of PDB are limited. Protection of the cluster by the PDB except for executing the `kubectl drain` command. Does the production environment need to enable PDB?

**Comment by travisn** [comment, marked as answer]
Wasn't this answered in #11174? A common scenario is that nodes are drained during node restarts or K8s upgrades. But if you don't ever drain nodes or don't want PDBs enabled, feel free to disable the feature in the cluster CR. But it's recommended for production clusters since they sometimes need to drain nodes.

**Reply by zeaone** [reply]
sorry, I didn't see #11174's answer yesterday, thanks for your answer，i got it.
